src/instructions/ping.rs

```diff
@@ -2,7 +2,7 @@ use super::{instruction_id, packet_id};
 use crate::bus::StatusPacket;
 use crate::{Bus, ReadError, TransferError, WriteError};
 
-#[derive(Debug)]
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Default)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Eq, PartialEq)]
```

(Also for my OCD ^^ )